examples/business_verification.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'smile-identity-core'
+require 'securerandom'
+# See https://docs.smileidentity.com/server-to-server/ruby/products/enhanced-kyc for
+# how to setup and retrieve configuation values for the IDApi class.
+
+# Initialize
+partner_id = '<Put your partner ID here>'; # login to the Smile Identity portal to view your partner id
+default_callback = '<Put your default callback url here>'
+api_key = '<Put your API key here>'; # copy your API key from the Smile Identity portal
+sid_server = '<0 | 1>'; # Use '0' for the sandbox server, use '1' for production server
+
+connection = SmileIdentityCore::BusinessVerification.new(partner_id, '', api_key, sid_server)
```

Comment:
we want to have this be called via the WebAPI for now.

adding product specific modules publicly is something for later.

Reply:
Will update the example to match this